dags/map_reproducibility/utils.py

```diff
@@ -38,13 +39,16 @@ def configure_project_and_cluster():
   return set_project_command
 
 
-# This is required to get auth to access
```

Comment:
let's leave the comment here as people may be curious about why we will need this dependency